### PR TITLE
feat(#major); uwu lend; add uwu lend

### PR DESCRIPTION
**Pull request by dmelotik**
Add UwU lend subgraph.

Testing: https://okgraph.xyz/?q=dmelotik%2Fuwu-lend-ethereum

borrow rewards broken still

### CODE REVIEW

File: subgraphs/aave-v2-forks/deploymentJsonContext.ts [hunk 1]
```diff
@@ -9,11 +9,11 @@ function slugToTitle(slug: string): string {
 }
 
 export const deploymentJsonContext: DeploymentJsonContextInterface = {
```
Comment (mrbrianhobo) [inline subgraphs/aave-v2-forks/deploymentJsonContext.ts:11]: i think this file is supposed to be git ignored

**Review by mrbrianhobo — APPROVED** [review]
uwu lgtm (⁄ ⁄>⁄ω⁄<⁄ ⁄)

File: subgraphs/aave-v2-forks/deploymentJsonContext.ts (hunk 1 above)
Comment (dmelotik) [reply subgraphs/aave-v2-forks/deploymentJsonContext.ts:11]: good catch, removing

**Comment by dmelotik** [comment]
Linting passes locally, going to merge

**Pull request merged**